library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
We can have parameters for jQuery, Bootstrap, KnockoutJS, datetimepicker, and fancybox. Anything more? We have to basically do this function for files inside `library/js`, right? Except for Bootstrap which is under assets..